airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3072,10 +3072,17 @@ components:
           type: string
 
         execution_date:
-          description: The execution date.
+          description: The execution date. Either set this or dag_run_id but not both.
           type: string
           format: datetime
```

Comment:
Do we want to deprecate this?

Reply:
I think we should continue to support it